client/src/Editor/App.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import AceEditor from "react-ace";
+
+import "ace-builds/src-noconflict/mode-css";
+import "ace-builds/src-noconflict/theme-monokai";
+
+class Editor extends React.Component {
+    constructor(props) {
```

Comment:
were you thinking of a child component where you will pass the props?  If so, this is fine.  If not, I suggest doing without props.